client/components/chart/index.js

```diff
@@ -211,12 +212,12 @@ class Chart extends Component {
 		chartHeight = width <= 783 ? 180 : chartHeight;
 		const legend = (
 			<Legend
-				className={ 'woocommerce-chart__legend' }
```

Comment:
Was this deliberate?  Does this break other chart legends?

Reply:
Yes, that class is not used anywhere.